src/pages/MyLots/MyLots.tsx

```diff
@@ -5,41 +5,39 @@ import { Container } from '../../shared/styled/containers/Container';
 import { PageHeader } from '../../shared/styled/headers/PageHeader';
 import Lot from './Lot/Lot';
 import { useTypedSelector } from '../../hooks/useTypedSelector';
-import { getLots } from '../../store/lots/lotsActions';
+import { createLot, getLots } from '../../store/lots/lotsActions';
 import { getParts } from '../../store/parts/partsActions';
 import { PrimaryButton } from '../../shared/styled/Elements/PrimaryButton';
+import { LotsModal } from './LotsModal';
+import { LotInterface } from '../../store/lots/types';
 
-export interface LotsInterface {
-  id: string;
-  category: string;
-  name: string;
-  description: string;
-  models: string[];
-  condition: string;
-  price: number;
-  photoUrl: string;
-}
-
+type SubmitBodyInterface = Omit<LotInterface, 'id'>;
 const MyLots: FC = () => {
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(getLots());
-  }, []);
+  }, [dispatch]);
   useEffect(() => {
     dispatch(getParts());
-  }, []);
+  }, [dispatch]);
```

Comment:
is it possible to combine two useEffects? like this:
```
useEffect(() => {
    dispatch(getLots());
    dispatch(getParts());
  }, []);
  }, [dispatch]);

Reply:
It possible. But I hear that is not good solution